nvflare/lighter/templates/aws_template.yml

```diff
@@ -0,0 +1,437 @@
+aws_start_sh: |
```

Comment:
if we move these files into "templates" we should remove them in "impl"?

Reply:
Yes, will do this in Part 2 of the PR. This is because the Dashboard is using this file now.